api/app/package.json

```diff
@@ -26,7 +26,7 @@
     "@aws-sdk/client-cloudwatch": "^3.338.0",
     "@medplum/core": "^2.0.18",
     "@medplum/fhirtypes": "^2.0.18",
-    "@metriport/api": "^4.1.7",
+    "@metriport/api": "^4.1.8",
```

Comment:
why is this getting bumped with a production version - if it's being released into develop?

Reply:
discussed offline - we'll release to prod to fix this asap